src/scandipwa/app/plugin/GTMHocWrapper.plugin.js

```diff
@@ -8,40 +8,31 @@ import {
     CMS_PAGE,
     HOME_PAGE,
     CUSTOMER_ACCOUNT
-} from 'Component/Header';
-
+} from 'Component/Header/Header.config';
 
 export const URL_REWRITE = 'url-rewrite';
 export const PASSWORD_CHANGE = 'password-change';
 export const CONFIRM_ACCOUNT = 'confirm_account';
 
 import withGTM from '../component/GoogleTagManager/withGTM.hoc';
 
-// TODO forgot password without HOC
-const homePage = Class => withGTM(Class, HOME_PAGE);
-const categoryPage = Class => withGTM(Class, CATEGORY);
-const searchPage = Class => withGTM(Class, SEARCH);
-const productPage = Class => withGTM(Class, PDP);
-const cmsPage = Class => withGTM(Class, CMS_PAGE);
-const cartPage = Class => withGTM(Class, CART);
-const checkout = Class => withGTM(Class, CHECKOUT);
-const passwordChangePage = Class => withGTM(Class, PASSWORD_CHANGE);
-const confirmAccountPage = Class => withGTM(Class, CONFIRM_ACCOUNT);
-const myAccount = Class => withGTM(Class, CUSTOMER_ACCOUNT);
-const menuPage = Class => withGTM(Class, MENU);
-const urlRewrites = Class => withGTM(Class, URL_REWRITE);
+const wrapperFunction = (route) => (args, callback, instance) => {
+    const originalRender = callback.apply(instance, args);
```

Comment:
Since some beta the context gets handled for you and all the callbacks are bound to the instances, so you can just write `callback(...args)` instead of `callback.apply(instance, args)`.
No harm from this tho, just for your convenience :')